Sources/Apodini/Request/Request.swift

```diff
@@ -5,28 +5,64 @@
 //  Created by Paul Schmiedmayer on 7/12/20.
 //
 
-import Vapor
+import Foundation
+import NIO
+@_implementationOnly import Vapor
 
+/// A type representing a server response.
+public protocol Request {
+    /// parameters
+    func parameter<T: Codable>(for parameter: Parameter<T>) throws -> T
+    /// The EventLoop associated with this request
+    var eventLoop: EventLoop { get }
+    /// database
+    var database: Any? { get }
+}
+
+typealias VaporRequest = Vapor.Request
+
+extension VaporRequest: Request {
+    public func parameter<T: Codable>(for parameter: Parameter<T>) throws -> T {
+
+        switch parameter.option(for: PropertyOptionKey.http) {
+        case .path where T.self is LosslessStringConvertible:
+            guard let parameter = parameters.get(parameter.id.uuidString) as? T else {
```

Comment:
The uuid is not the actual name...

Reply:
This is undefined right now since we are not yet handling PathParameters in the RESTSemantic model builder.